helm/defectdojo/values.yaml

```diff
@@ -250,8 +250,14 @@ django:
     # could be emptyDir (not for production) or pvc
     type: emptyDir
     # in case if pvc specified, should point to the already existing pvc
-    persistentVolumeClaim: media
-
+    persistentVolumeClaim:
+      # set to true if django.mediaPersistentVolume.type is set to pvc
```

Comment:
isn't "# set to true to create a new pvc and if django.mediaPersistentVolume.type is set to pvc" correct?

Reply:
Yes it's ok to have it the way you put it above. I will update it accordingly.